docs/content/dagster-cloud/deployment/hybrid.mdx

```diff
@@ -1,7 +1,74 @@
 ---
-title: "Hybrid deployment in Dagster Cloud | Dagster Docs"
-
+title: Hybrid deployment in Dagster Cloud | Dagster Docs
 noindex: True
 ---
 
 # Hybrid deployment in Dagster Cloud
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+In this guide, we'll cover how Hybrid deployment works in Dagster Cloud, including its system architecture, agent, and security.
+
+---
+
+## Hybrid architecture overview
+
+A **hybrid deployment** utilizes a combination of your infrastructure and Dagster-hosted backend services.
```

Comment:
Maybe in a follow up diff, but we should have or link to a section on hybrid vs serverless